datalake/backend/processing/check_stac_metadata/task.py

```diff
@@ -5,13 +5,17 @@
 from json import dumps, load
 from os import environ
 from os.path import dirname, join
-from typing import Callable, List, TextIO
+from typing import Callable, Iterable, List, MutableMapping, TextIO
 from urllib.parse import urlparse
 
 import boto3
 from botocore.response import StreamingBody
 from jsonschema import Draft7Validator, FormatChecker, RefResolver, ValidationError
 from jsonschema._utils import URIDict
+from pynamodb.attributes import UnicodeAttribute
+from pynamodb.models import Model
+
+PROCESSING_ASSETS_TABLE_NAME = "processing_assets"
```

Comment:
utils.ResourceName enum contains this string

Reply:
Only applicable once https://github.com/linz/geospatial-data-lake/pull/198 merges